js/views/userPageVw.js

```diff
@@ -322,15 +318,29 @@ module.exports = baseVw.extend({
     });
   },
 
-  bindModalCloseHandler: function(e) {
-    if (this.modalCloseHandler) return;
-
-    this.modalCloseHandler = function() {
-      window.history.back();
+  loadingStatus: function() {
```

Comment:
Here's where I declare the loading config for the user page. Since the user page is the same view for both the User's Page and the detail of a Listing, I've added some logic to send back a different config depending on which flavor of the page we're trying to connect to.